.pre-commit-config.yaml

```diff
@@ -10,3 +10,15 @@ repos:
     hooks:
     -   id: flake8
         args: [--extend-ignore=E501]
+
+  - repo: local
+    hooks:
+      - id: bicep
+        name: bicep
+        description: Lint and build Bicep files
+        entry: ./infra/generate_arm_template.sh
+        language: script
+        files: \.bicep$
+        require_serial: true
+        args: # Bicep files that we want to generate ARM templates from
+          - -f=./infra/main.bicep
```

Comment:
Will this cause the main.bicep file to be converted to ARM even if it wasn't changed?
e.g. if I modify `keyvault.bicep`

Is that what we want? I assume so, but want to check

Reply:
Here - I have changed a word in a child template and it has still regenerated the json
![image](https://github.com/microsoft/rag-experiment-accelerator/assets/6254153/ac47c991-64b4-49ed-836d-9e57b462a1a6)